### DISCUSSION TITLE
I am going to write a game for Atari Jaguar using a custom-made Assembler to optimize performance and C++ for game logic.  Do you think this is a bad idea?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#109466] [Programming Help]
### Body

Please reply

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DeandrePetersen** [comment]
Whether it's a good idea depends on your specific goals, requirements, and constraints. If performance optimization is critical and you have the necessary expertise in both assembly language and C++, it could be a viable approach. However, if you're concerned about development time, portability, or maintainability, you may want to consider alternatives or carefully weigh the trade-offs.
